k8s.gcr.io/images/k8s-staging-sig-storage/generate.sh

```diff
@@ -46,6 +46,7 @@ readonly images=(
 for image in "${images[@]}"; do
     echo "- name: ${image}"
     echo "  dmap:"
+    podman run --rm --volumes-from gcloud-config gcr.io/google.com/cloudsdktool/cloud-sdk:latest \
```

Comment:
Can you explain what this does? Not all devs are going to have podman as their container runtime.

Reply:
Sorry, that wasn't supposed to be part of the PR.